package task01;

public class MainException extends Exception{

	public MainException() {
		System.out.println("잘못된 입력입니다");
	}
	
}
